etc/adminhtml/system/adyen_giving.xml

```diff
@@ -66,7 +66,7 @@
             <label>Suggested donation amounts</label>
             <backend_model>Adyen\Payment\Model\Config\Backend\DonationAmounts</backend_model>
             <config_path>payment/adyen_giving/donation_amounts</config_path>
-            <tooltip><![CDATA[Suggested amounts that the shopper can choose to donate. The minimum amount is 1 EUR.]]></tooltip>
+            <tooltip><![CDATA[Suggested amounts that the shopper can choose to donate. Amount should be specified in minor units, e.g 100 = 1 EUR. The minimum amount is 1 EUR.]]></tooltip>
```

Comment:
I think that asking for minor units here can be confusing, is there something we could do to format it when it is about to be used? Using `\Adyen\Payment\Helper\Data::formatAmount` for example.

Reply:
addressed in 0c4e426